data-hibernate-jpa/src/test/groovy/io/micronaut/data/hibernate/AbstractHibernateQuerySpec.groovy

```diff
@@ -71,6 +71,14 @@ abstract class AbstractHibernateQuerySpec extends AbstractQuerySpec {
     @Inject
     RelPersonRepository relPersonRepo
 
+    void "test where in empty list of entities"() {
+        when:
+            def found = ratingRepository.findByAuthors(List.of())
```

Comment:
The test is failing because `ratingRepository` does not have `findByAuthors(List)` method.
You can add this method `public abstract List<Book> findByAuthorInList(List<Author> authors);` to the `BookRepository` and then update test and maybe add case to find in non empty list
```
void "test where in empty list of entities"() {
        when:
        def found = bookRepository.findByAuthorInList(List.of())
        then:
        found.empty
        when:
        def author = authorRepository.findByName("Stephen King")
        found = bookRepository.findByAuthorInList(List.of(author))
        then:
        !found.empty
    }
```

Reply:
Oh goodness that's embarrassing, I promise it did fail when I did define that function 😅 I tried to make two separate branches, one with the test and one with the test and fix and I think I lost the repository changes in that shuffle

Reply:
I know, it happens when need to fork and create PR from new branch there.

Reply:
Updated to use BookRepository instead to utilize existing seed data